<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regist</class>
 <widget class="QWidget" name="regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>填写用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameinput_LE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>填写密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pswdinput_LE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>重复密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rpswdinput_LE"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="regist_PB">
     <property name="text">
      <string>确认注册</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
